lms/static/sass/course/_info.scss

```diff
@@ -103,8 +103,9 @@ div.info-wrapper {
     }
 
     ol {
+      margin: 0 0 14px 26px;
```

Comment:
I would put padding on the container itself, so anything - not just `ol` or `ul` would look correct. Let's move the padding to the parent container - the sidebar as a whole.